packages/ruby/spec/http_response_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "http_response"
+
+RSpec.describe HttpResponse do
+  describe "#body" do
+    it "returns an empty string when the body is empty" do
+      response = HttpResponse.from_parts(200, {}, nil)
+
+      expect(response.body).to eq ""
+    end
+
+    it "concatenates the body when it is StringIO" do
+      body = "BODY"
+      response = HttpResponse.from_parts(200, {}, StringIO.new(body))
+
+      expect(response.body).to eq body
+    end
+  end
```

Comment:
I wonder if we also need a test around rewinding of IOs, similar to https://github.com/readmeio/metrics-sdks/blob/master/packages/ruby/spec/http_request_spec.rb#L157-L166?

Reply:
Added this in my latest commit 👍